### ISSUE TITLE
Filtred categories appear on search

### ISSUE THREAD

**Issue by Fzwael** [joeattardi/emoji-button#37]
Hello,

I was using your component from the last couple of days and it's perfect 👍 

One small thing I saw is that when I created a picker with some categories, let's say this :
``` typescript
    const picker = new EmojiButton({
      categories: ['smileys', 'animals', 'objects', 'people']
    });
```

I still can see other categories when searching (for example flags) : 
![image](https://user-images.githubusercontent.com/11984138/76411551-54f5ee80-6392-11ea-96d9-718c24dd46af.png)

This could be confusing from an user end.

**Comment by joeattardi** [comment]
Whoops!

Thanks for finding this, I will get it fixed ASAP.

**Comment by joeattardi** [comment]
Fixed in v2.12.1!
